.github/workflows/docs-and-linting.yml

```
@@ -1,8 +1,11 @@
 name: Render and Lint Documentation
 
 on:
+  push:
+    branches: [ 'main' ]
+    tags: [ 'v*.*.*' ]
   pull_request:
-    branches_ignore: []
+    branches: [ 'main' ]
```

Is this heavy enough to be worth gating?  Why not simply let it run on all pushes and all PRs?

```suggestion
  push:
  pull_request:
```

(this is perhaps more of a question for @sudo-bmitch than @ScottBrenner, to be clear :heart:)

---

The downside of running on all pushes and PRs is if a maintainer pushes directly to a branch rather than their own fork, you end up with multiple runs of the same job. I'm not opposed to running on all PRs, but I think that's solving a problem we don't have since we don't have feature branches in the spec today, and I don't think many people are reviewing the generated artifacts anyway.